en/docs/administration/configure-a-custom-domain-for-apis.md

```diff
@@ -0,0 +1,48 @@
+# Configure a Custom Domain for APIs
+
+A custom domain for your site is essential for effective branding, discoverability, and credibility. Choreo allows you to configure a custom domain for the APIs you deploy in it.
+
+To understand how to configure a custom domain for your APIs, follow the steps in the following sections:
+
+## Prerequisite
+
+To configure a custom domain for your APIs, you must first register a custom domain with a domain registrar service as follows:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google/ GitHub/ Microsoft account.
```

Comment:
shall we say, using Gogole, GitHub, or Microsoft

Reply:
Done